src/openrct2-ui/windows/EditorScenarioOptions.cpp

```diff
@@ -1030,49 +1030,47 @@ static void window_editor_scenario_options_guests_paint(rct_window* w, rct_drawp
         // Cash per guest label
         x = w->windowPos.x + 8;
         y = w->windowPos.y + w->widgets[WIDX_CASH_PER_GUEST].top;
-        gfx_draw_string_left(dpi, STR_CASH_PER_GUEST_LABEL, nullptr, COLOUR_BLACK, x, y);
+        gfx_draw_string_left(dpi, STR_CASH_PER_GUEST_LABEL, nullptr, COLOUR_BLACK, { x, y });
 
         // Cash per guest value
         x = w->windowPos.x + w->widgets[WIDX_CASH_PER_GUEST].left + 1;
         y = w->windowPos.y + w->widgets[WIDX_CASH_PER_GUEST].top;
         arg = gGuestInitialCash;
-        gfx_draw_string_left(dpi, STR_CURRENCY_FORMAT_LABEL, &arg, COLOUR_BLACK, x, y);
-    }
-
-    // Guest initial happiness label
-    x = w->windowPos.x + 8;
-    y = w->windowPos.y + w->widgets[WIDX_GUEST_INITIAL_HAPPINESS].top;
-    gfx_draw_string_left(dpi, STR_GUEST_INIT_HAPPINESS, nullptr, COLOUR_BLACK, x, y);
+        gfx_draw_string_left(dpi, STR_CURRENCY_FORMAT_LABEL, &arg, COLOUR_BLACK, { x, y });
 
-    // Guest initial happiness value
-    x = w->windowPos.x + w->widgets[WIDX_GUEST_INITIAL_HAPPINESS].left + 1;
-    y = w->windowPos.y + w->widgets[WIDX_GUEST_INITIAL_HAPPINESS].top;
-    arg = (gGuestInitialHappiness * 100) / 255;
-    gfx_draw_string_left(dpi, STR_PERCENT_FORMAT_LABEL, &arg, COLOUR_BLACK, x, y);
-
-    // Guest initial hunger label
-    x = w->windowPos.x + 8;
-    y = w->windowPos.y + w->widgets[WIDX_GUEST_INITIAL_HUNGER].top;
-    gfx_draw_string_left(dpi, STR_GUEST_INIT_HUNGER, nullptr, COLOUR_BLACK, x, y);
+        // Guest initial happiness label
```

Comment:
Why did you move all these blocks into the `if (w->widgets[WIDX_CASH_PER_GUEST].type != WWT_EMPTY)` block?